### PR TITLE
[Ready for review] Polish tranlsation WAI wideo introduction

### PR DESCRIPTION
**Pull request by zwiastunsw**
- video-metadata.yml modified
- video-introduction/video-introduction.pl.vtt added

related: https://github.com/w3c/wai-video-standards-and-benefits/pull/133

### CODE REVIEW

File: video-metadata.yml [hunk 1]
```diff
@@ -259,6 +259,7 @@
   name:
     en: "Introduction to Web Accessibility and W3C Standards"
     fr: "Introduction à l’accessibilité Web et aux standards du W3C"
+    pl: "Wprowadzenie do dostępnośc internetowej i standardów W3C"
```
Comment (jcabak) [inline video-metadata.yml:262]: dostępności

**Review by jcabak — APPROVED** [review]

**Review by jcabak — APPROVED** [review]

**Comment by zwiastunsw** [comment]
thx

**Review by jcabak — APPROVED** [review]

**Comment by remibetin** [comment]
@jcabak Thank you for your contribution. Have you had the chance to see my message here? https://github.com/w3c/wai-video-standards-and-benefits/pull/133#issuecomment-2042220550

**Comment by remibetin** [comment]
QA: done ✅

**Pull request merged**